src/form/acp.jl

```diff
@@ -129,6 +131,21 @@ function constraint_ohms_yt_to{T <: AbstractACPForm}(pm::GenericPowerModel{T}, f
 end
 
 """
+Creates Ohms constraints for DC Lines (yt post fix indicates that Y and T values are in rectangular form)
+
+```
+(1-loss1) * p_fr + (p_to - loss0 * br_status) == 0
+```
+"""
+function constraint_ohms_yt_dc{T <: AbstractACPForm}(pm::GenericPowerModel{T}, f_bus, t_bus, f_idx, t_idx, br_status, loss0, loss1)
```

Comment:
A better name for this constraint would be something like `constraint_dc_line`.  If you have a good taxonomy for different dc line formulations, maybe a few extra characters to indicate where this formulation fits in that taxonomy.